.github/workflows/ci-core.yml

```diff
@@ -165,6 +165,7 @@ jobs:
   # Inclusive check: all (new) modules are analyzed, but no need to enable "required" checks for each one
   golangci-matrix-results-validation:
     name: lint
+    if: ${{ always() }}
```

Comment:
This failed out of the queue because the linter jobs were skipped:
![image](https://github.com/user-attachments/assets/cb00acb0-75c2-415b-b353-bc97e9ce7fe6)
So do we need another condition here?
cc @chudilka1

Reply:
or we need to permit `skipped` alongside `success`?